applications/GeoMechanicsApplication/custom_processes/apply_c_phi_reduction_process.hpp

```diff
@@ -0,0 +1,204 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors: Marjan Fathian
+//                Wijtze Pieter Kikstra
+//                Anne van de Graaf
+//
+
+#pragma once
+
+// System includes
+#include <cmath>
+#include <iostream>
+
+// Project includes
+#include "custom_elements/small_strain_U_Pw_diff_order_element.hpp"
+#include "custom_utilities/stress_strain_utilities.h"
+#include "includes/element.h"
+#include "includes/model_part.h"
+#include "utilities/math_utils.h"
+
+// Application includes
+#include "geo_mechanics_application_variables.h"
+#include "includes/kratos_flags.h"
+#include "includes/kratos_parameters.h"
+#include "processes/process.h"
+
+namespace Kratos
+{
+
+class ApplyCPhiReductionProcess : public Process
```

Comment:
This class is not templated, so we should split this into a .h and .cpp file. Then we need to add the export, i.e:
```suggestion
class KRATOS_API(GEO_MECHANICS_APPLICATION) ApplyCPhiReductionProcess : public Process
```

Reply:
split